packages/amplify-provider-awscloudformation/lib/upload-appsync-files.js

```diff
@@ -43,16 +43,76 @@ async function uploadAppSyncFiles(context, resourcesToUpdate, allResources, opti
     return deploymentRootKey;
   };
 
+  const getApiKeyConfigured = () => {
+    const projectDetails = context.amplify.getProjectDetails();
```

Comment:
The duplicated code was eliminated since it was moved to a method where no ```projectDetails``` digging needed since we already have the AppSync API available filtered, so this snippet became a non-duplicate.